dbt/scripts/utils/town_active_schedule.csv.example

```diff
@@ -0,0 +1 @@
+township_code,township_name,tri_code,tri_name,active_start_date,active_end_date

```

Comment:
The fact that we'd have to keep this schedule up-to-date based on Valuations' QC requirements is the biggest downside of this approach, I think. Still, it strikes me as better than producing reports for every township every day, since the query load would actually be quite extensive if we wanted to do that.

One alternative might be to refactor the way that the `export_qc_town_close_reports.py` script works so that it queries data for all towns in one Athena query and then splits up the data for export by township in Python. This might actually be a more sustainable solution from an operational perspective, but since it's a bigger refactor, I'm curious to hear what you think.

Reply:
Having looked through this a little bit more, I actually like this first approach more than the second one. It's nice to be able to run a single town-level report ad-hoc, which the second option wouldn't really support.

Reply:
**suggestion (non-blocking):** May want to add an example line in here, just to demo the format of dates and such.

Reply:
I've refactored the design to move away from the activity schedule approach, so this point is moot now.